### PR TITLE
Add update resource + Config map resource

### PR DESCRIPTION
**Pull request by ILpinto**
Add update to resource https://github.com/openshift/openshift-restclient-python#replace
Add ConfigMap resource

### CODE REVIEW

File: resources/resource.py [hunk 1]
```diff
@@ -218,3 +218,11 @@ def _extract_data_from_yaml(self, yaml_data):
         self.name = yaml_data.get('metadata').get('name')
         self.api_version = yaml_data.get('apiVersion')
         self.kind = yaml_data.get('kind')
+
+    def update(self, resource_dict):
+        """
+        Update resource with resource dict
+        :param resource_dict: Resource dict
```
Comment (myakove) [inline resources/resource.py]: Use google doc style, please.

File: resources/resource.py [hunk 2]
```diff
@@ -218,3 +218,11 @@ def _extract_data_from_yaml(self, yaml_data):
         self.name = yaml_data.get('metadata').get('name')
         self.api_version = yaml_data.get('apiVersion')
         self.kind = yaml_data.get('kind')
+
+    def update(self, resource_dict):
```
Comment (myakove) [inline resources/resource.py]: logs?

File: resources/resource.py (hunk 1 above)
Comment (ILpinto) [reply resources/resource.py]: DONE

File: resources/resource.py (hunk 2 above)
Comment (ILpinto) [reply resources/resource.py]: Added

**Pull request merged**